modules/dastool/scaffolds2bin/main.nf

```diff
@@ -0,0 +1,47 @@
+include { initOptions; saveFiles; getSoftwareName; getProcessName } from './functions'
+
+params.options = [:]
+options        = initOptions(params.options)
+
+process DASTOOL_SCAFFOLDS2BIN {
+    tag "$meta.id"
+    label 'process_low'
+    publishDir "${params.outdir}",
+        mode: params.publish_dir_mode,
+        saveAs: { filename -> saveFiles(filename:filename, options:params.options, publish_dir:getSoftwareName(task.process), meta:meta, publish_by_meta:['id']) }
+
+    conda (params.enable_conda ? "bioconda::das_tool=1.1.3" : null)
+    if (workflow.containerEngine == 'singularity' && !params.singularity_pull_docker_container) {
+        container "https://depot.galaxyproject.org/singularity/das_tool:1.1.3--r41hdfd78af_0"
+    } else {
+        container "quay.io/biocontainers/das_tool:1.1.3--r41hdfd78af_0"
+    }
+
+    input:
+    tuple val(meta), path(fasta)
+    val(binner)
```

Comment:
binner could be moved inline with the input tuple as I assume you would want it on a per-contig set basis.